src/Constants/authorization.js

```diff
@@ -0,0 +1,117 @@
+/* 
```

Comment:
Nitpick: I wouldn't consider 'constants' to be the right directory for this. Maybe call the folder 'http' or 'spotify' since it's handling all of the stuff you need to interact with spotify?

Reply:
Done